lib/hashvec.cpp

```diff
@@ -314,11 +314,12 @@ void hash_vector_base::redo_hash() {
         if (erased[i]) continue;
         const void *k = getkey(i);
         if (find(k, &cache)) {
-            if (info->ismulti)
-                while (find_next(k, &cache))
-                    ;
-            else
+            if (info->ismulti) {
```

Comment:
These changes are a little funky, I believe explicit brackets make the intent clearer.

Reply:
Yes!